documentation/release_notes.markdown

```diff
@@ -4,116 +4,104 @@ title: "Puppet Server: Release Notes"
-2016-03-09 11:48:17,460 ERROR [main] [p.t.internal] Error during app buildup!
-java.lang.RuntimeException: Service ':VersionedCodeService' not found
-```
+### Bug fix: Implement DELETE request handling on the `certificate_reqest` endpoint
 
-To resolve this, add this line to your `bootstrap.cfg`:
+Unlike the Ruby Puppet master, previous versions of Puppet Server didn't handle [DELETE requests to the `certificate_request` endpoint](https://docs.puppet.com/puppet/latest/reference/http_api/http_certificate_status.html#delete). Server 2.4.0 resolves this by handling these requests the same way that the Ruby master does.
 
-```
-puppetlabs.services.versioned-code-service.versioned-code-service/versioned-code-service
-```
+-   [SERVER-977](https://tickets.puppetlabs.com/browse/SERVER-977)
 
-Alternatively, you can merge your changes into the new version of `bootstrap.cfg` (the `bootstrap.cfg.rpmnew` file in the above example) and replace `bootstrap.cfg` with the new file.
+### Bug fixes: Certificate status endpoint behaviors
 
-* [SERVER-1058](https://tickets.puppet.com/browse/SERVER-1058)
+Puppet Server 2.4.0 resolves these issues with the [`certificate_status` endpoint](https://docs.puppet.com/puppet/latest/reference/http_api/http_certificate_status.html):
 
-#### Puppet Server fails to start when configured to log to syslog
+-   **Handle nil values in `desired_state` more gracefully ([SERVER-542](https://tickets.puppetlabs.com/browse/SERVER-542)):** If the `desired_state` of a PUT request to the `certificate_status` endpoint was nil, previous versions of Server threw a NullPointerException. Server 2.4.0 resolves this issue.
+-   **Respect asterisks in `certificate_statuses` requests ([SERVER-864](https://tickets.puppetlabs.com/browse/SERVER-864)):** Previous versions of Server wouldn't return a list of certificates to authenticated `certificate_statuses` requests if the request included an asterisk (`*`). Server 2.4.0 resolves this issue.
 
-If its Logback service is configured to log to syslog, Puppet Server 2.3.0 fails to start. Puppet Server 2.3.1 fixes this regression, which did not affect prior versions of Puppet Server.
+### Other issues
 
-* [SERVER-1215](https://tickets.puppet.com/browse/SERVER-1215)
+-   **Remove hyphens in `puppet-server`:** We've [changed the name of our GitHub repository](https://tickets.puppetlabs.com/browse/SERVER-1206) from `puppet-server` to `puppetserver` and [removed the hyphen from many other references](https://tickets.puppetlabs.com/browse/SERVER-392).
+-   **Log Ruby backtraces ([SERVER-1273](https://tickets.puppetlabs.com/browse/SERVER-1273)):** Previous versions of Server didn't log Ruby backtraces. Server 2.4.0 does, just like a Ruby Puppet master.
+-   **Don't override the service startup timeout ([SERVER-557](https://tickets.puppetlabs.com/browse/SERVER-557)):** Previous versions of Server 2.x overrode the OS-specific service startup timeout with a value of 120 seconds. Server 2.4.0 removes this override.
```

Comment:
Should we also mention what the new timeout would be as a result of removing the override?  EZBake's current default is now 5 minutes.